### PR TITLE
Fixes #16637 - host applicable packages ui

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-hosts.module.js [hunk 1]
```diff
@@ -240,15 +240,33 @@ angular.module('Bastion.content-hosts').config(['$stateProvider', function ($sta
...
+    })
+    .state('content-hosts.details.packages.actions', {
+        url: '/packages/actions',
+        permission: 'edit_hosts',
+        collapsed: true,
+        controller: 'ContentHostPackagesActionsController',
+        templateUrl: 'content-hosts/content/views/content-host-packages-actions.html'
+    })
+    .state('content-hosts.details.packages.installed', {
+        url: '/packages/installable',
```
Comment (jlsherrill) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-hosts.module.js]: done

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-packages-applicable.controller.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * @ngdoc object
+ * @name  Bastion.content-hosts.controller:ContentHostPackagesApplicableController
+ *
+ * @requires $scope
+ * @resource $timeout
+ * @resource $window
+ * @requires HostPackage
+ * @requires translate
+ * @requires Nutupane
+ * @requires BastionConfig
```
Comment (daviddavis) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-packages-applicable.controller.js]: I think you're missing `@requires Package` here. Also, BastionConfig isn't used.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-packages-installed.controller.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.content-hosts.controller:ContentHostPackagesController
+ *
+ * @requires $scope
+ * @resource $timeout
+ * @resource $window
+ * @requires ContentHostPackage
```
Comment (daviddavis) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-packages-installed.controller.js]: s/ContentHostPackage/HostPackage/

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-packages-installed.controller.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.content-hosts.controller:ContentHostPackagesController
```
Comment (daviddavis) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-packages-installed.controller.js]: Should be ContentHostPackagesInstalledController

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-packages-applicable.html [hunk 5]
```diff
@@ -0,0 +1,65 @@
+<span page-title ng-model="host">{{ 'Packages for: ' | translate }} {{ host.name }}</span>
+
+<h4 translate>Applicable Packages</h4>
+
+<div data-extend-template="layouts/details-nutupane.html">
+
+  <span data-block="no-rows-message" translate>
+    The host has not reported any applicable packages for upgrade..
```
Comment (daviddavis) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-packages-applicable.html]: Double period

**Comment by jlsherrill** [comment]
@daviddavis addressed comments

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-packages-applicable.html [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+  </span>
+
+  <span data-block="no-search-results-message">
+    Your search returned zero Packages.
+  </span>
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-packages-applicable.html]: This should be marked for translation.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-packages-applicable.html [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+          <button class="btn btn-default dropdown-toggle"
+                  ng-hide="!remoteExecutionPresent || denied('edit_hosts', contentHost)"
+                  ng-disabled="working ||  detailsTable.numSelected === 0"
+                  type="button" id="use-remote-execution" data-toggle="dropdown">
+            <span class="caret"></span>
+          </button>
+          <ul class="dropdown-menu" role="menu" aria-labelledby="use-remote-execution">
+            <li role="presentation"><a ng-click="performViaKatelloAgent('packageUpdate', getKatelloAgentCommand())" role="menuitem" tabindex="-1" href="#" translate>via Katello agent</a></li>
+            <li role="presentation"><a ng-click="performViaRemoteExecution('packageUpdate', getRemoteExecutionCommand(), false)" role="menuitem" tabindex="-1" href="#" translate>via remote execution</a></li>
+            <li role="presentation"><a ng-click="performViaRemoteExecution('packageUpdate', getRemoteExecutionCommand(), true)" role="menuitem" tabindex="-1" href="#" translate>via remote execution - customize first</a></li>
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-packages-applicable.html]: Same comment as above about readability and these long lines.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-packages-installed.html [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+<h4 translate>Installed Packages</h4>
+
+<div data-extend-template="layouts/details-nutupane.html">
+  <span data-block="no-rows-message" translate>
+    The host has not reported any installed packages, registering with subscription-manager should cause these to be reported.
+  </span>
+
+  <span data-block="no-search-results-message">
+    Your search returned zero Packages.
+  </span>
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-packages-installed.html]: This needs to be marked for translation.

**Comment by waldenraines** [comment]
Note to self: reload before review.

**Pull request merged**